lua/ttt2/extensions/net.lua

```diff
@@ -6,58 +6,157 @@
 local net = net
 local util = util
 local table = table
-local tostring = tostring
 local isfunction = isfunction
 
 -- Stream network message name constant
 local NETMSG_STREAM = "TTT2_NET_STREAM"
+local NETMSG_REQUEST_NEXT_SPLIT = "TTT2_NET_REQUEST_NEXT_SPLIT"
 
 if SERVER then
 	AddCSLuaFile()
 
 	-- Add the network string for streaming data
 	util.AddNetworkString(NETMSG_STREAM)
+	-- Add the network string for confirming single splits
+	util.AddNetworkString(NETMSG_REQUEST_NEXT_SPLIT)
 end
 
 -- Size to split the network stream at (currently a bit lower than the max value, just to have some buffer)
+-- Can be up to 65.533KB see: https://wiki.facepunch.com/gmod/net.Start
 net.STREAM_FRAGMENTATION_SIZE = 65400
 
 -- Stream cache variables
+local wait_stream_cache = {}
+local eligible_Players  = {}
```

Comment:
is there a reason they are local and not part of net like the two below?

Reply:
Only Idea that comes to mind is, that those are handled in another way on a reload. 
Maybe I did that, because saving them in net is unnecessary.

Not sure whats better or if there was a reason.

Reply:
First, lowercase player in the variable name.

Second, id say it would be better to also keep those one the table just because the others are too.

Reply:
We changed that to the net-table